README.md

```diff
@@ -35,9 +35,11 @@ with PodmanClient(base_url=uri) as client:
 
     # find all containers
     for container in client.containers.list():
-        first_name = container['Names'][0]
-        container = client.containers.get(first_name)
+        # it is important to get the container object again with the get()
+        # method to get all the fields like container.status populated
+        container = client.containers.get(container.name)
```

Comment:
```suggestion
        container.reload()
```
would also populate all the fields.

PS. The comment could also include a notice that sparse=True is our default. Future work could allow sparse=False on the list() call.

Reply:
thanks @jwhonce, good idea. I committed new changes